index.html

```diff
@@ -3,111 +3,106 @@
 title: Your New Jekyll Site
 ---
 
-<div id="info" class="clearfix">
+<section class="info">
 
-    <div id="screens">
-        <a href="/screenshots/" title="view more screenshots of i3">
-        <img id="screenshot" src="/screenshots/i3-9.bigthumb.png" width="375" height="234" alt="i3 screenshot"
-	 srcset="/screenshots/i3-12.bigthumb.jpg 2x">
+    <div class="info-text">
+        <h2>Do What I Mean. Good Docs. Clean Code. Sounds good?</h2>
+        <p>
+            Then you will love i3. Watch the <a href="screenshots/">screencast</a>,
+            read the <a href="docs/userguide.html">User’s Guide</a> or <a href="downloads/">install i3</a>!
+        </p>
+        <a class="download" href="downloads">
+            <span class="info-arrow">➡</span>
+            Download the latest version
+            <span class="version">4.19</span>
         </a>
     </div>
 
-    <h2>Do What I Mean. Good Docs. Clean Code. Sounds good?</h2>
-    <p>
-    Then you will love i3. Watch the <a href="/screenshots/">screencast</a>,<br>
-    read the <a href="/docs/userguide.html">User’s Guide</a> and <a
-    href="/downloads/">install i3</a>!
-    </p>
-    <div id="download">
-        <a href="/downloads">
-        <span style="font-weight: bold; color: #3A8ECD; margin-right: .5em">➡</span>
-        Download the latest version
-        <span style="margin-left: 2em; color: #c0c0c0">4.19</span>
-        </a>
-    </div>
-</div>
+    <a class="screen" href="screenshots/" title="view more screenshots of i3">
+        <img src="screenshots/i3-14.bigthumb.png" alt="i3 screenshot"
```

Comment:
Can you stick with the current screenshot please? The one visible right now does not use the default config (on the contrary, it visibly deviates from the default), and I don’t want to give people the wrong impression about what they’re going to get once they install i3.

Reply:
242b610